lib/assert.js

```diff
@@ -47,8 +47,10 @@ class AssertionError extends Error {
 		// Reserved for power-assert statements
 		this.statements = [];
 
-		if (opts.stack) {
+		if (typeof opts.stack === 'string') {
```

Comment:
I'd think this would no longer be the case. Is it still needed?

Reply:
I was looking at [this](https://github.com/avajs/ava/blob/master/lib/test.js#L280) when including that condition.